### ISSUE TITLE
Enable support for widescreen horizontal thumbnails in a library

### ISSUE THREAD

**Issue by roostercactus** [jellyfin-archive/jellyfin-roku-legacy#901]
**Is your feature request related to a problem? Please describe.**
This is a request to add widescreen horizontal thumbnail support in a library.  Some video thumbnails are widescreen (horizontal) in shape, versus the vertical poster thumbnails (like for Movies).  When particular thumbnails are displayed currently they are cropped inside a vertical poster shaped thumbnail.

**Describe the solution you'd like**
Add the ability for widescreen horizontal thumbnails to be displayed in their original format (OAR) without being cropped into a vertical poster shaped thumbnail.

**Describe alternatives you've considered**
None at the moment.

**Additional context**
Even though I'm using a mixed library I setup my directory layout according to the 'Shows' content type.  So in my case the folder structure looks like this:  

Media/Channel/Uploads/video.mkv
Media/Channel/Uploads/video.jpg

which parallels a 'Show' type setup:

Shows/Show/Season/Episode01.mkv
Shows/Show/Season/Episode01.jpg

The Jellyfin web-client is able to display the thumbnails in their original aspect ratio and format.  Here are some screen shots comparing the same library viewed via the web-client versus the Roku client.

You can see in the Roku screen shots the thumbnails are cropped inside a vertical poster formatted thumbnail.

Roku client 'Shows/Channels' screen
![roku_client_1_shows](https://user-images.githubusercontent.com/22780470/208126597-af6e8144-d47a-4f53-8d07-9625798bcf68.jpg)

Roku client 'Seasons/Uploads' screen
![roku_client_2_seasons](https://user-images.githubusercontent.com/22780470/208126688-3a10e8cc-7b5e-45e8-b2ef-1901bb896a7c.jpg)

Roku client 'Episodes/Videos' screen
![roku_client_3_episodes](https://user-images.githubusercontent.com/22780470/208126761-dbaabbfe-4025-489c-a04e-ea9ef3111bd8.jpg)

Jellyfin web-client 'Episodes/Videos' screen for comparison which shows the original widescreen format that is desired in the Roku client
![jellyfin_web_client_episodes](https://user-images.githubusercontent.com/22780470/208126846-fe1998b7-daf1-4f15-88ad-d5f3a5258714.jpg)

**Comment by roostercactus** [comment]
These are the thumbnails I download along with each video...

https://yewtu.be/vi/<video_id>/maxresdefault.jpg

for example:

https://yewtu.be/vi/N02OuWlA8w8/maxresdefault.jpg

In case that helps.  They are 1280x720 resolution.

**Comment by sevenrats** [comment]
you may get better results on this request if you rephrase it to honor the project's piracy policy. youtube ripping is not a supported use-case for jellyfin.  if the devs do this it will because they want consistency between the web app and the roku app.
